src/main/groovy/com/morpheusdata/nutanix/prismelement/plugin/NutanixPrismElementPluginProvisionProvider.groovy

```diff
@@ -565,6 +565,7 @@ class NutanixPrismElementPluginProvisionProvider extends AbstractProvisionProvid
 			maxStorage    : maxStorage,
 			networkConfig : workloadRequest.networkConfiguration,
 			rootVolume    : rootVolume,
+			snapshotId : opts.snapshotId
```

Comment:
logic was already in place to create a server from snapshot, the `snapshotId` was just missing